From 9921446bf69a57858e989ba988bd9e712110312f Mon Sep 17 00:00:00 2001
From: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
Date: Fri, 25 Aug 2023 11:46:25 +0530
Subject: [PATCH] wifi: nl80211: Changing the nl80211_fam structure as per the
 latest openwrt

Adding code changes in the nl80211_fam structure as in latest openwrt
there is additional ops added in the nl80211 commands.

Signed-off-by: Gautham Kumar Senthilkumaran <quic_gauthamk@quicinc.com>
---
 net/wireless/nl80211.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/wireless/nl80211.c b/net/wireless/nl80211.c
index 40922a3..edbfa6a 100644
--- a/net/wireless/nl80211.c
+++ b/net/wireless/nl80211.c
@@ -18410,7 +18410,7 @@ static struct genl_family nl80211_fam __genl_ro_after_init = {
 	.small_ops = nl80211_small_ops,
 	.n_small_ops = ARRAY_SIZE(nl80211_small_ops),
 #if LINUX_VERSION_IS_GEQ(6,1,0)
-	.resv_start_op = NL80211_CMD_REMOVE_LINK_STA + 5,
+	.resv_start_op = NL80211_CMD_REMOVE_LINK_STA + 6,
 #endif
 	.mcgrps = nl80211_mcgrps,
 	.n_mcgrps = ARRAY_SIZE(nl80211_mcgrps),
-- 
2.17.1

